pennylane_lightning_gpu/src/algorithms/AdjointDiffGPU.hpp

```diff
@@ -1081,7 +494,9 @@ template <class T = double> class AdjointJacobianGPU {
                     std::size_t offset_first, std::size_t offset_last) {
                     // Ensure No OpenMP threads spawned;
                     // to be resolved with streams in future releases
+                    // omp_set_num_threads(offset_last - offset_first +1);
                     omp_set_num_threads(1);
```

Comment:
Do we need to set the number of threads as `offset_last - offset_first +1` as shown in line 497.

Reply:
Nope, I'll remove the comment until we get support for streams.